README.md

```diff
@@ -124,6 +124,97 @@ If you want to check what resources are going to be targeted without actually te
 cloud-nuke aws --resource-type ec2 --dry-run
 ```
 
+### Config file
+
+For more granularity, you can pass in a configuration file to specify which resources to terminate using regular expressions.
```

Comment:
NIT to fix minor ambiguity in sentence:

```suggestion
For more granularity, such as being able to specify which resources to terminate using regular expressions, you can pass in a configuration file.
```

Reply:
Also, consider adding a sentence that the config file is a new feature, and only a few types of more granular filters are supported, but we'll be adding more in the future, and PRs to help with this endeavor are very welcome!

Reply:
- [ ] Also, consider adding a sentence that the config file is a new feature, and only a few types of more granular filters are supported, but we'll be adding more in the future, and PRs to help with this endeavor are very welcome!